backend/src/database/migrations/20220408162426_create_suppliers_table.js

```diff
@@ -0,0 +1,13 @@
+exports.up = function (knex) {
+  return knex.schema.createTable("suppliers", function (table) {
+    table.increments(); //id
+    table.string("name").notNullable();
```

Comment:
this one is complicated, but have we decided to use name as oppose to first/last or some version of this.

Reply:
It hasn't been decided upon, no.  We should discuss further and make a decision.